packages/terra-abstract-modal/tests/wdio/abstract-modal-spec.js

```diff
@@ -115,6 +115,16 @@ describe('Abstract Modal', () => {
     Terra.should.beAccessible();
   });
 
+  describe('Custom Focusable Fallback', () => {
+    before(() => browser.url('/#/raw/tests/terra-abstract-modal/abstract-modal/abstract-modal-fallback-focus'));
+
+    it('clicks to open modal', () => {
+      browser.click('button');
+    });
+
+    Terra.should.matchScreenshot({ selector: 'div[role="dialog"]' });
```

Comment:
Is there a way to determine that the focus fallback element actually has focus? Webdriver.io has this command: http://webdriver.io/api/state/hasFocus.html

Reply:
I'm not sure. This is mostly iOS this issue is appearing up, as that doesn't focus on the modal like it does on desktop chrome.

Reply:
It worked. https://github.com/cerner/terra-core/pull/2112/commits/66602e5d456163d54d146893b281b7a046ee74b3